### PR TITLE
Document Risco options flow

### PR DESCRIPTION
**Pull request by OnFreund**
## Proposed change

Add documentation describing the options in the Risco options flow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/39177
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by OnFreund** [comment]
The parent for this is already merged

File: source/_integrations/risco.markdown [hunk 1]
```diff
@@ -27,6 +27,17 @@ An Alarm Control Panel entity will be created for each partition in your site, a
...
 
+### Extra configuration of the integration
+
+You can configure additional options through the integration options flow:
+
+| Option | Description |
+| -------| ----------- |
+| How often to poll Risco (in seconds) | The lower this is, the faster your entities will reflect changes, but the more resource intensive it'll be. |
+| Require pin code to arm | When checked, you'll need to enter your pin code when arming through Home Assistant. |
+| Require pin code to disarm | When checked, you'll need to enter your pin code when disarming through Home Assistant. |
```
Comment (frenck) [inline source/_integrations/risco.markdown]: We have helpers for this now. Example:

```
{% configuration_basic %}
The name of the option:
  description: "Description blablabal"
Other option:
  description: Lorem ipsum
{% endconfiguration_basic %}
```

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/risco.markdown (hunk 1 above)
Comment (OnFreund) [reply source/_integrations/risco.markdown]: Thanks. Like this?

Comment (frenck) [reply source/_integrations/risco.markdown]: Yep!

image

File: source/_integrations/risco.markdown [hunk 2]
```diff
@@ -27,6 +27,19 @@ An Alarm Control Panel entity will be created for each partition in your site, a
 
 If you have multiple sites, only the first site will be used.
 
+### Extra configuration of the integration
```
Comment (frenck) [inline source/_integrations/risco.markdown]:
```suggestion
## Options
```

File: source/_integrations/risco.markdown [hunk 3]
```diff
@@ -27,6 +27,19 @@ An Alarm Control Panel entity will be created for each partition in your site, a
...
+
+You can configure additional options through the integration options flow:
```
Comment (frenck) [inline source/_integrations/risco.markdown]: Please describe how to get there. "options flow" is a developer term.

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks,

**Review by frenck — APPROVED** [review]

**Comment by OnFreund** [comment]
Thanks @frenck!

**Pull request merged**